packages/ui/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { MouseEventHandler } from 'react';
+import { forwardRef, HTMLChakraProps, Image as ChakraImage } from '@chakra-ui/react';
+
+import blackHorizontalLogo from '../../../images/LogoBlackHorizontal.svg';
+import blackSquareLogo from '../../../images/LogoBlackSquare.svg';
+import whiteHorizontalLogo from '../../../images/LogoWhiteHorizontal.svg';
+import whiteSquareLogo from '../../../images/LogoWhiteSquare.svg';
+
+type Layout = 'horizontal' | 'square';
+
+type LogoColor = 'white' | 'black';
+
+export interface LogoProps extends HTMLChakraProps<'img'> {
+  layout?: Layout;
+  color?: LogoColor;
+  onClick?: MouseEventHandler;
+}
+
+const logosMap = {
+  white: { horizontal: whiteHorizontalLogo as unknown as string, square: whiteSquareLogo as unknown as string },
```

Comment:
`as unknown as string` doesn't seem right. Why did you need to add them?

Reply:
it's a typecheck thing - it says that StaticImageData cannot be cast to string